packages/web-components/src/components/va-select/va-select.tsx

```diff
@@ -173,8 +178,15 @@ export class VaSelect {
   }
 
   render() {
-    const { error, reflectInputError, invalid, label, required, name, hint, uswds } = this;
+    const { error, reflectInputError, invalid, label, required, name, hint, messageAriaDescribedby, uswds } = this;
 
+    const errorID = uswds ? 'input-error-message': 'error-message';
```

Comment:
I don't see a reason to keep these error message IDs different. Should we update this to be the same across both versions? Or not worth the effort?

Reply:
I would say not worth the effort, we'll be removing the V1 versions of components in a couple months anyways.